Nicer formatting of example notebook in the docs

jch1g10 opened the pull request:
Closes #22 

### I have:

- [x] Formatted any Python files with [black](https://github.com/ambv/black)
- [x] Brought the branch up to date with master
- [x] Added any relevant Github labels
- [x] Added tests for any new additions
- [x] Added or updated any relevant documentation
- [ ] Added an Architectural Decision Record (ADR), if appropriate
- [ ] Added an [MPLv2 License Header](https://www.mozilla.org/en-US/MPL/headers/) if appropriate 

### Description

I have added headings and comments to `example_usage.ipynb`, and formatted the outputs of Jupyter notebook cells as blockquotes when they appear in the docs. I have also added a `get_status` function to the client, as a more human-friendly function for checking the status of a query.

@greenape

greenape commented:
Didn't we just take the demo folder _out_ in #4?

maxalbert commented:
Nice additions, thank you! For the time being I suggest only keeping a single version of the example usage notebook because the existing docs one and the new demo one are fairly similar (the only difference is how flowclient is installed and how the token is generated). I expect we will evolve the existing one into a more full walkthrough in the near future anyway.

jch1g10 commented:
> Nice additions, thank you! For the time being I suggest only keeping a single version of the example usage notebook because the existing docs one and the new demo one are fairly similar (the only difference is how flowclient is installed and how the token is generated). I expect we will evolve the existing one into a more full walkthrough in the near future anyway.

Good suggestion. I've removed the demo notebook, and updated the title to better describe what this pull request is actually doing.

maxalbert commented on docs/source/analyst/example_usage.ipynb:
```diff
@@ -103,36 +185,41 @@
      "data": {
       "text/plain": [
-       "(False, <Response [202]>)"
+       "(True, <Response [303]>)"
```
Isn't the output of `get_status()` supposed to be a string now? Looks like some of the cells need to be re-executed maybe?

jch1g10 replied:
Yes, that output was left over from before I added `get_status()`. I've cleared all outputs now, and checked that the output of `get_status()` is correct when it runs during the docs build.

greenape commented on docs/source/analyst/example_usage.ipynb:
```diff
@@ -314,23 +402,33 @@
    "source": [
-    "dl1 = flowclient.daily_location(date=\"2016-01-01\", aggregation_unit=\"admin3\", daily_location_method=\"last\", subscriber_subset=None)\n",
-    "dl2 = flowclient.daily_location(date=\"2016-01-02\", aggregation_unit=\"admin3\", daily_location_method=\"last\", subscriber_subset=None)"
+    "# dl1 = flowclient.daily_location(\n",
```
Hmm. Actually, can we either remove this cell, or uncomment it?

jch1g10 replied:
Good point. I think it's useful to show there are two ways to create a modal location, so I've kept this cell in and uncommented it.
